stream2hop/Utils.py

```diff
@@ -9,7 +10,9 @@ def __init__(self, scimmaUrl, scimmaConfFile):
         self.msgCount = 0
 
     def open(self):
-        self.stream = io.Stream(config=self.scimmaConfFile, format="json")
+        # self.stream = io.Stream(config=self.scimmaConfFile, format="json")
+        self.stream = io.Stream(format="json")
+
```

Comment:
Is this intended? Looks like it's ignoring the configuration now.

Reply:
Opps not intended at all, I was trying on my own server with no configurations. will fix it